apps/tsdashboard/wristbandprinter.py

```diff
@@ -13,29 +13,55 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
+
+# Notes:
+# Printing from the raspberry pi uses the CUPS driver solution.
+# General Info on the Zebra ZD510-HC can be found at:
+# https://www.zebra.com/us/en/support-downloads/printers/desktop/zd510-hc.html
+# There is a specific link for information on the CUPS package.
+#
+# Once installed, the driver needs to be configured for this printer:
+# Size: custom
+# Width = 1
+# Heigth = 7.024 (May need to change for different sizes.
+# Units: Inches
+# Resolution: 300dpi
+# Meadia Tracking: Non-continuous (Mark sensing)
+# Mesia Type: Printer default
+#
+# In my .bashrc file, I also set this:
+# export PRINTER="Zebra_Technologies_ZTC_ZD510-300dpi_ZPL"
+#
+# The application uses LPR, so this export is required.
+
+
+
 class WristBandPrinter:
     def __init__(self):
         pass
 
     def getUserFormattedWristband(self):
         return self.__str__();
-        
+
+# Because the second line cut off the first few characters,
+# place six spaces after the "\n" on the beginning of the second line.
+
     def __str__(self):
         val = "{}{} {}{} {}{} {}{} {}{} {}{} {}{} {}".format(
         str(self._patientData.id) if self._idShow else "",
-        "\n" if self._idnl else "",
+        "\n      " if self._idnl else "",
         self._patientData.first if self._firstShow else "",
-        "\n" if self._firstnl else "",
+        "\n      " if self._firstnl else "",
         self._patientData.middle if self._middleShow else "",
-        "\n" if self._middlenl else "",
+        "\n      " if self._middlenl else "",
```

Comment:
Probably we want a flag here, self._middlenl is here to add a new line, some additional flag is needed for spaces.

Reply:
If want a variable containing 6 spaces instead of 6 spaces, it may be easier to modify in the future, but I think this will always be 6 spaces. The wristband "grows" to the right if longer. A three line print would be subject to the same thing.